.github/workflows/ci.yml

```diff
@@ -1,38 +1,293 @@
+      - uses: ./.github/actions/build-sample-app
+        with:
+          app-dir: 'dataflow-website/stream-developer-guides/feature-guides/streams/function-composition/composed-http-transformer-kotlin-rabbitmq'
+          mvn-build-commands: '-B clean install jib:dockerBuild'
+          artifactory-publish: ${{ inputs.maven-build-only != 'true' }}
+          jf-artifactory-spring: ${{ secrets.JF_ARTIFACTORY_SPRING }}
+          docker-push: ${{ inputs.maven-build-only != 'true' }}
+          docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
+          docker-password: ${{ secrets.DOCKERHUB_TOKEN }}
+          docker-images: >-
+            springcloudstream/composed-http-transformer-kotlin-rabbitmq:0.0.1-SNAPSHOT
+
+  task-demo-metrics-prometheus:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/build-sample-app
+        with:
+          app-dir: 'monitoring-samples/task-apps/task-demo-metrics-prometheus'
+          mvn-build-commands: '-B clean install docker:build'
+          artifactory-publish: ${{ inputs.maven-build-only != 'true' }}
+          jf-artifactory-spring: ${{ secrets.JF_ARTIFACTORY_SPRING }}
+          docker-push: ${{ inputs.maven-build-only != 'true' }}
+          docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
+          docker-password: ${{ secrets.DOCKERHUB_TOKEN }}
+          docker-images: >-
+            springcloudtask/task-demo-metrics-prometheus:0.0.4-SNAPSHOT
+
+  scenario-task:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/build-sample-app
+        with:
+          app-dir: 'scenario-task'
+          mvn-build-commands: '-B clean install spring-boot:build-image -Dspring-boot.build-image.imageName=springcloudtask/scenario-task:0.0.1-SNAPSHOT'
+          artifactory-publish: ${{ inputs.maven-build-only != 'true' }}
+          jf-artifactory-spring: ${{ secrets.JF_ARTIFACTORY_SPRING }}
+          docker-push: ${{ inputs.maven-build-only != 'true' }}
+          docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
+          docker-password: ${{ secrets.DOCKERHUB_TOKEN }}
+          docker-images: >-
+            springcloudtask/scenario-task:0.0.1-SNAPSHOT
+
+  timestamp-task:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/build-sample-app
+        with:
+          app-dir: 'timestamp-task'
+          mvn-build-commands: '-B clean install spring-boot:build-image -Dspring-boot.build-image.imageName=springcloudtask/timestamp-task:1.0.0'
+          artifactory-publish: ${{ inputs.maven-build-only != 'true' }}
+          jf-artifactory-spring: ${{ secrets.JF_ARTIFACTORY_SPRING }}
+          docker-push: ${{ inputs.maven-build-only != 'true' }}
+          docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
+          docker-password: ${{ secrets.DOCKERHUB_TOKEN }}
+          docker-images: >-
+            springcloudtask/timestamp-task:1.0.0
```

Comment:
Did we want to make the task samples 2.0.0?

Reply:
I was going to do that in https://github.com/spring-cloud/spring-cloud-dataflow-samples/issues/156 step 5. However, is it a problem if these get published as is and then follow up and get published under 2.0.0. I am sure I have aleady published them. under 1.0.0 w/ my testing of the workflows.